ManatiCR/vitaminau
# VIT-78: Beneficiary changes forms.

# Task [VIT-78](https://manati.atlassian.net/browse/VIT-78).
Changes in beneficiaries section in all forms.

# Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the following URL: 
  - http://vitaminau.docker/formularios/venta-de-productos
  - http://vitaminau.docker/formularios/proyecto-cultural-deportivo
    - Make the "Inversión" field not required. Add Help Text "Indique el monto en colones. Si no posee inversión, colocar 0."
  - http://vitaminau.docker/formularios/tecnologias-licenciables
- Replace the name of the field "Correo electrónico de la UCR" by "Correo para solicitar información"
- In the "Beneficiarios" section, in the "Persona individual" field, when selecting "Con especificaciones" you must display a text area "Especificaciones".
- In the section of "Beneficiarios" when selecting "Comunidad" the text area "Detalle comunidad" should have the help text "Escriba el nombre de los distritos u otras señas."
- In the "Beneficiarios" section, when selecting "Comunidad" and selecting any province, you must display a help text at the top where the cantons are displayed that says "Para seleccionar múltiples cantones mantenga presionada la tecla ctrl (si está en Linux o Windows) o la tecla command (si está en Mac OS)” y de click en los cantones que necesita."

# CA
- Check change label name "Correo electrónico de la UCR" to "Correo para solicitar información".
- Check the help text in "inversión" and now it's no required.
- Check beneficiaries:
  - Markup help text in cantons and help text in community detail.
  - Options in individual person change -> Specific person to Specifications and the text area change to Specifications.
